src/csharp/Grpc.Reflection/ReflectionGrpc.cs

```diff
@@ -63,58 +63,6 @@ public abstract partial class ServerReflectionBase
 
     }
 
-    /// <summary>Client for ServerReflection</summary>
-    public partial class ServerReflectionClient : grpc::ClientBase<ServerReflectionClient>
```

Comment:
Client types are not in Grpc.Core.Api.

After I created this PR it occurred to me that this client is public and could be used by users of this package.